.ci/packer_cache.sh

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+# This script takes responsibility to create a minikube cluster. It has all
+# of the necessary default settings so that no environment variable has to
+# be specified.
+
+declare -a docker_images=("docker.elastic.co/eck/eck-ci:57e169ff162c2ea351a1956e1ad355b9")
```

Comment:
What's the update process? Do we need to manually change the tag here?
Why not computing the tag value automatically here?

Reply:
@jmlrt When `packer_cache.sh` is executed do we have an access to the git repo content?

Reply:
Yes, the full repository is cloned the `packer_cache.sh` is executed from inside the repository directory ([ansible/roles/packer_cache/tasks/linux/packer_cache.yml#L26-L39](https://github.com/elastic/infra/blob/master/ansible/roles/packer_cache/tasks/linux/packer_cache.yml#L26-L39))

Reply:
@jmlrt On the machine where it will be executed do we have `make`, `md5sum` and `awk`?

Reply:
I can confirm you that they are already installed when `packer_cache.sh` script is run.

Reply:
Great news :)

Reply:
Fixed. It will use makefile to get tag for image, same as on CI